### ISSUE TITLE
Integrate heat timeseries in etrago tables

### ISSUE THREAD

**Issue by IlkaCu** [openego/eGon-data#583]
Integrate heat timeseries in etrago tables by extending [hts_etrago.py](https://github.com/openego/eGon-data/blob/dev/src/egon/data/datasets/heat_etrago/hts_etrago.py). 
Depends on #582

**Comment by ClaraBuettner** [comment]
I am not very sure which task is meant here. In case it was meant to be for eGon100RE (which I guess), it was solved in powerd-data.

**Comment by ClaraBuettner** [comment]
Done in powerd-data which was merged into `dev`
